man/ruby.1

```diff
@@ -438,20 +438,22 @@ Show long help message same as
 check of syntax same as
 .Fl c
 .Fl -yydebug
+.Pp
+.El
+.Pp
+.Bl -hang -offset indent
+Or one of the following, which are intended for debugging the interpreter:
 .It Sy yydebug
 compiler debug mode, same as
 .Fl -yydebug
-.Pp
-Only specify this switch if you are going to debug the Ruby interpreter.
 .It Sy parsetree
+Print a textual representation of the Ruby AST for the program
 .It Sy parsetree_with_comment
-AST nodes tree
-.Pp
-Only specify this switch if you are going to debug the Ruby interpreter.
+Print the same tree as parsetree, but with each node annotated with the program code
```

Comment:
I'm confused as to what

> annotated with the program code

means. From what I can tell, `parsetree_with_comment` is `parsetree` with comments.

Reply:
yah. I was trying to communicate _what_ the comments were (they're the annotated Ruby source corresponding to the parse tree node). I'll re-word this to make it more obvious